starting_from_scratch_lab/lab_40/tar2.js

```diff
@@ -0,0 +1,5 @@
+const selectElement = document.querySelector(".source");
+selectElement.addEventListener("change", (event) => {
```

Comment:
`keyup` would be a better event than `chnage` since it is executed right as you type, so you can see all the boxes change at once.